README.md

```diff
@@ -36,13 +36,24 @@ The contents of this repository are licensed under the [Apache License 2.0](./LI
 To get involved, please see [Code of Conduct](./CODE_OF_CONDUCT.md) and [contribution guidelines](./CONTRIBUTING.md).
 
 # About Eiffel
-This repository forms part of the Eiffel Community. Eiffel is a protocol for technology agnostic machine-to-machine communication in continuous integration and delivery pipelines, aimed at securing scalability, flexibility and traceability. Eiffel is based on the concept of decentralized real time messaging, both to drive the continuous integration and delivery system and to document it.
+This repository forms part of the Eiffel Community. Eiffel is a protocol for 
+technology agnostic machine-to-machine communication in continuous integration 
+and delivery pipelines, aimed at securing scalability, flexibility and 
+traceability. Eiffel is based on the concept of decentralized real time 
+messaging, both to drive the continuous integration and delivery system and to 
+document it.
 
-Visit [Eiffel Community](https://eiffel-community.github.io) to get started and get involved.
+Visit [Eiffel Community](https://eiffel-community.github.io) to get started and 
+get involved.
 
 # Documentation
 
-1. [**GUI Overview**](./wiki/GUI-Overview.md)
-1. [**Subscription Handling**](./wiki/Subscription-Handling.md)
-    -  [**Add Subscription**](./wiki/Add-Subscription.md)
-1. [**Test Rules User Guide**](./wiki/Test-Rules.md)
+1. [**GUI Overview**](wiki/markdown/GUI-Overview.md)
+1. [**Subscription Handling**](wiki/markdown/Subscription-Handling.md)
+    -  [**Add Subscription**](wiki/markdown/Add-Subscription.md)
+1. [**Test Rules**](wiki/markdown/Test-Rules.md)
+    - [**Rule testing mechanism via "Test Rules" GUI interface**](wiki/markdown/Test-Rules.md#Rule-testing-mechanism-via-Test-Rules-GUI-interface)
+    - [**Introduction**](wiki/markdown/Test-Rules.md#Introduction)
```

Comment:
Those sub headers was removed by a PR before this, should they really be added back?

Reply:
Fixed now